frontend/src/components/common/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+
+type Props = {
+  id: string;
+  icon: string;
+  onAction: (id: string) => void;
+};
+
+const IconButton: FC<Props> = ({ id, icon, onAction }) => {
+  const handleAction = (): void => {
+    onAction(id);
+  };
+
+  return (
+    <button className={styles.button} onClick={handleAction}>
+      <img src={icon} alt="Action" />
+    </button>
```

Comment:
```suggestion
    <Button props />
```
our button component

Reply:
It's irrational to use a `<Button> `which anyway returns a simple 
`<button><img><button>`
in the end, also it requires a label now and has styles that I won't use. 
As for me IconButton and LinkButton must be extended from Button and be on different levels. If LinkButton is inside of the Button, _why then not put IconButton also in there?_ 

**The answer is because it is overloading and it is better and simpler to use separate IconButton than change Button, add there more props, styles and conditions**
So, why don't I just use a separate IconButton component?

Reply:
we should use our common `Button` component for `IconButton` 👍

Reply:
I hadn't seen an answer to my question and will ask straight. Why should we use our common Button component here?

Reply:
because we can do it 👍